apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLogger.cs

```diff
@@ -163,8 +164,8 @@ private SourceLocation CreateSourceLocation(Exception exception)
             }
 
             StackTrace stackTrace = new StackTrace(exception, true);
-            StackFrame[] frames = stackTrace.GetFrames();
-            if (frames.Length == 0)
+            StackFrame[] frames = stackTrace?.GetFrames();
```

Comment:
This change seems unnecessary. `stackTrace` cannot be null since it's created on the previous line.